lib/components/intent.js

```diff
@@ -535,6 +535,47 @@ Intent.prototype.setPresence = function(presence, status_msg=undefined) {
     });
 };
 
+/**
+ * @typedef {
+ *       "m.event_not_handled"
+ *     | "m.event_too_old"
+ *     | "m.internal_error"
+ *     | "m.foreign_network_error"
+ *     | "m.event_unknown"
+ * } BridgeErrorReason
+ */
+
+/**
+ * Signals that an error occured while handling an event by the bridge.
+ * @param {string} roomId ID of the room in which the error occured.
+ * @param {string} eventId ID of the event for which the error occured.
+ * @param {string} network_name Name of the bridged network.
+ * @param {BridgeErrorReason} reason The reason why the bridge error occured.
+ * @param {string} affected_users Regex matching all affected users.
+ * @return {Promise}
+ */
+Intent.prototype.signalBridgeError = function(
+    roomId,
+    eventId,
+    network_name,
+    reason,
+    affected_users
+) {
+    return this.sendEvent(
+        roomId,
+        "de.nasnotfound.bridge_error",
+        {
+            "network_name": network_name,
```

Comment:
You don't need to quote JSON in Javascript, it's better without. Also you can use shorthand here for nearly all of these names like:

```
{
   network_name,
   reason,
   affected_users, //NOTE: You may want to validate this one
   "m.relates_to": { stuff in here ...}
}
```